### ISSUE TITLE
problème d'emoji sur le brownie

### ISSUE THREAD

**Issue by pinage404** [laem/futureco-data#54]
problème d'emoji avec le brownie

![Screen Shot 2019-11-03 at 15 22 52](https://user-images.githubusercontent.com/6325757/68086601-d5896900-fe4d-11e9-9743-d17d4cf1f7fb.png)

https://futur.eco/simulateur/nourriture/Brownie-%E2%80%91-aux-noix-enti%C3%A8re

**Comment by laem** [comment]
Yep je crois que j'ai utilisé quelques emojis pas encore universels.

**Comment by laem** [comment]
J'ai corrigé pour le brownie, par contre le beurre ne passe pas. J'ai l'impression que notre version de twemoji n'est pas à jour, car sur emojipedia on voit que twitter a bien un emoji pour le beurre

**Comment by pinage404** [comment]
https://futur.eco/simulateur/nourriture/Soupe-%C3%A0-l'oignon j'ai que la soupe

![Screen Shot 2019-11-07 at 23 33 49](https://user-images.githubusercontent.com/6325757/68433542-11f2f700-01b7-11ea-8e18-38b4e65438eb.png)
